### PR TITLE
Work-around broken python-apt pypi package on xenial.

### PR DESCRIPTION
**Pull request by bac**

### CODE REVIEW

**Comment by bac** [comment]
QA:

On clean trusty and xenial LXDs:

```
make clean sysdeps setup
```

**Comment by bac** [comment]
@frankban This will solve your problem. Feel free to *shipit* before I arrive tomorrow if it helps you.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/123/
Test PASSed.

File: test-requirements.pip [hunk 1]
```diff
@@ -33,7 +33,7 @@ PyYAML==3.10
 
 # Charm hooks.
 launchpadlib==1.10.2
-python-apt==0.8.5
+#python-apt==0.8.5
```
Comment (frankban) [inline test-requirements.pip]: Add a comment about why it's commented out, or remove this line?

**Comment by frankban** [comment]
👍  with a minor comment and QA ok.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui-charm/124/
Test PASSed.

**Comment by jrwren** [comment]
👍

**Comment by mhilton** [comment]
:+1:

**Comment by bac** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-charm-merge

**Pull request merged**